io_import_psd_layers_as_planes.py

```diff
@@ -425,13 +448,13 @@ def create_textured_plane(name, transforms, global_matrix, import_id, layer_inde
         else:
             bbox = bboxes[i]
             transforms = get_transforms(layer, bbox, i_offset)
```

Comment:
Shouldn't this line be removed?

Reply:
Hi Jasper!
Actually, I added `get_dimensions`, which is called both by `get_transforms` and elsewhere.
Splitting `get_dimensions` is only useful as part of commit `f231873`, which is an ongoing attempt to optionally get the original PSD coordinates in an additional UV layer, but it isn't complete yet :/
(This would be useful for applying other textures to all objects at once, from the whole psd, in my case, normal maps.)

Reply:
Oops, seems I didn't have a thorough look... 😊 
Are you also coming to Bconf with Flavio again?

Reply:
Yeah, totally! We're doing a conference as well this year, your name might just be in it ;)

Reply:
😬 Nice, see you there!